src/com/dakl/Main.java

```diff
@@ -194,7 +200,9 @@ public static Space parseSpace(String piece_str) {
     }
 }
 
+// REVIEW : separate files for classes
 class CheckerBoard {
+    // REVIEW: member variables
```

Comment:
The convention for member variables is for them to be private, and have this class be in charge of providing the values and modifying the values (principle of encapsulation).